### DISCUSSION TITLE
Date de sortie de la prochaine version 3.5.1

### DISCUSSION THREAD

**Discussion by Aurely9n** [JeromeDevome/GRR#218] [General]
Bonjour,

Auriez-vous une idée de la date approximative de quand sortira la prochaine version release de GRR 3 (la v3.5.1 par exemple) ? Il y a plusieurs évolutions depuis la v3.5.0a qui m'intéressent et j'aimerai quantifier le bénéfice d'installer et d'utiliser une version encore développement plutôt que d'attendre la sortie de la prochaine version.

Très cordialement,
Aurélien

**Comment by ynaessens** [comment]
Bonjour,
la RC ne devrait plus trop tarder et selon les retours des aimables testeurs, la stable suivra.
Cordialement,
YN

**Reply by Aurely9n** [reply]
Merci pour votre retour.

Cordialement,
Aurélien
